source/about/self-hosted-subscriptions.rst

```diff
@@ -62,7 +73,13 @@ To add more users to your existing subscription, `contact us <https://mattermost
 
 When you subscribe to an annual plan with Mattermost, you agree to provide quarterly reports of the actual number of registered users within your system if you're above the amount you subscribed in your purchase order. A registered user is a user who has an account in a workspace and does not show as *Inactive* in **System Console > User Management > Users**.
 
-We'll send you an email notice around the end of the quarter reminding you to send us your report. To send this report, take a screenshot of **System Console > Site Statistics** and send in a reply email back to us. Please ensure your screenshot is taken from the top of the page and includes the **Total Active Users** metric.
+We'll send you an email notice around the end of the quarter reminding you to send us your report. 
+
+From Mattermost v7.7, you can send us this information from **System Console > License and Edition > True-up Review**. 
+
+If your deployment doesn't have internet access, you'll need to export a bundle which is uploaded via the Customer Portal. It's then accessed by the Custome Success team. In **System Console > True-up Review**, select **Export Bundle**. Then log into your Customer Portal account, navigate to the **True-up Review** section and upload the bundle.
```

Comment:
@justinegeffen actually since we found out the effort here is considerable, we decided for v1 for customers to export the bundle and then send it to their CSM via email

Reply:
Updated, thanks!